server/odc-migrate/src/main/resources/migrate/common/V_4_3_3_3__iam_role_permission.yaml

```diff
@@ -0,0 +1,66 @@
+kind: resource
```

Comment:
will we release this on public aliyun? if so, there may be some transaction problems

Reply:
Yes, it will. Any suggestion for the transaction problem?